workers/deployment_manager/DeploymentManager/DeploymentService/DeploymentModifier.cs

```diff
@@ -0,0 +1,210 @@
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using LogLevel = Improbable.Worker.LogLevel;
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.SpatialOS.Platform.Common;
+using Improbable.SpatialOS.Snapshot.V1Alpha1;
+using System.Security.Cryptography;
+using System;
+using System.IO;
+using System.Net;
+
+namespace DeploymentManager
+{
+    public class DeploymentModifier
+    {
+        private static DeploymentServiceClient deploymentServiceClient;
+        private static SnapshotServiceClient snapshotServiceClient; 
+
+        public static void Init(string token)
+        {
+            var credentials = new PlatformRefreshTokenCredential(token);
+            deploymentServiceClient = DeploymentServiceClient.Create(credentials: credentials);
+
+            snapshotServiceClient = SnapshotServiceClient.Create(credentials: credentials);
+        }
+
+        public static string UploadSnapshot(string snapshotPath, string projectName, string deploymentName)
+        {
+            // Read snapshot.
+            var bytes = File.ReadAllBytes(snapshotPath);
+
+            if (bytes.Length == 0)
+            {
+                Console.Error.WriteLine($"Unable to load {snapshotPath}. Does the file exist?");
```

Comment:
Pretty sure if the file doesn't exist the method will throw 😉

Reply:
oh that whole log is wrong 🤦‍♀️ 
that happens when you copy-paste from other parts